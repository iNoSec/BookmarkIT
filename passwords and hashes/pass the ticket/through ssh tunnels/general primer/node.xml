<?xml version="1.0" encoding="UTF-8"?>
<node>
<version>6</version>
<dict>
  <key>title</key><string>General Primer</string>
  <key>nodeid</key><string>62cd0152-31eb-4775-83a7-c1d011033f8e</string>
  <key>modified_time</key><integer>1528829286</integer>
  <key>version</key><integer>6</integer>
  <key>content_type</key><string>text/xhtml+xml</string>
  <key>created_time</key><integer>1528829187</integer>
  <key>order</key><integer>0</integer>
</dict>
</node>
