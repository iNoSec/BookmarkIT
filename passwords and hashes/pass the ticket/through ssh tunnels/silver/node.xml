<?xml version="1.0" encoding="UTF-8"?>
<node>
<version>6</version>
<dict>
  <key>title</key><string>Silver</string>
  <key>nodeid</key><string>c240e273-814d-47c6-bf89-3b179da2055a</string>
  <key>modified_time</key><integer>1528829046</integer>
  <key>version</key><integer>6</integer>
  <key>content_type</key><string>text/xhtml+xml</string>
  <key>created_time</key><integer>1528828726</integer>
  <key>order</key><integer>2</integer>
</dict>
</node>
