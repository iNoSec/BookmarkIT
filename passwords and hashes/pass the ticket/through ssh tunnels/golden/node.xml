<?xml version="1.0" encoding="UTF-8"?>
<node>
<version>6</version>
<dict>
  <key>title</key><string>Golden</string>
  <key>nodeid</key><string>3ece7d9c-5071-4f84-818f-f0442f1039ed</string>
  <key>modified_time</key><integer>1528828716</integer>
  <key>expanded2</key><true/>
  <key>version</key><integer>6</integer>
  <key>content_type</key><string>text/xhtml+xml</string>
  <key>created_time</key><integer>1528828403</integer>
  <key>order</key><integer>1</integer>
</dict>
</node>
